src/Controls/tests/TestCases.HostApp/Issues/Issue28986_ScrollView.xaml

```diff
@@ -0,0 +1,99 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Maui.Controls.Sample.Issues.Issue28986_ScrollView"
+             x:Name="TestPage"
+             Title="SafeArea ScrollView Test">
+    
+    <Grid x:Name="MainGrid" BackgroundColor="Blue" SafeAreaEdges="All">
```

Comment:
In order for this test to work this will need to be set to "None" otherwise the SCrollView won't be able to go edge to edge